junit-platform-console/src/main/java/org/junit/platform/console/ConsoleLauncherExecutionResult.java

```diff
@@ -50,8 +54,8 @@ static ConsoleLauncherExecutionResult failed() {
 		return new ConsoleLauncherExecutionResult(FAILED);
 	}
 
-	static ConsoleLauncherExecutionResult forSummary(TestExecutionSummary summary) {
-		return new ConsoleLauncherExecutionResult(summary);
+	static ConsoleLauncherExecutionResult forSummary(TestExecutionSummary summary, CommandLineOptions options) {
+		return new ConsoleLauncherExecutionResult(summary, options);
```

Comment:
I think we should just have a single constructor `ConsoleLauncherExecutionResult(exitCode, summary)`, inline `computeExitCode()` into `forSummary`, and change the unit tests to use `forSummary`. Could be done in a subsequent commit/PR, though.